[INFO   ] Logger: Record log in /storage/emulated/0/Download/Python/.kivy/logs/kivy_24-04-13_86.txt
[INFO   ] Kivy: v2.2.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.11.4 (main, Sep 30 2023, 10:54:38) [GCC 11.4.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 190 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Audio: Providers: audio_sdl2 (audio_android, audio_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 v1.r32p1-01eac0.8135628637558d6d27e8ec61da0ffbb5'>
[INFO   ] GL: OpenGL vendor <b'ARM'>
[INFO   ] GL: OpenGL renderer <b'Mali-G68 MC4'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16383>
[INFO   ] GL: Texture max units <128>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] Base: Unknown <android> provider
[INFO   ] Base: Start application main loop
[INFO   ] GL: NPOT texture support is available
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Download/Python/main.py", line 32, in <module>
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/app.py", line 956, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/base.py", line 574, in runTouchApp
[WARNING] stderr:     EventLoop.mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/base.py", line 339, in mainloop
[WARNING] stderr:     self.idle()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/base.py", line 379, in idle
[WARNING] stderr:     Clock.tick()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/clock.py", line 733, in tick
[WARNING] stderr:     self.post_idle(ts, self.idle())
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/clock.py", line 776, in post_idle
[WARNING] stderr:     self._process_events()
[WARNING] stderr:   File "kivy/_clock.pyx", line 620, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 653, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 649, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 218, in kivy._clock.ClockEvent.tick
[WARNING] stderr:   File "/storage/emulated/0/Download/Python/game.py", line 60, in update
[WARNING] stderr:     self.winer_label.text = "Player 2 Won"
[WARNING] stderr:     ^^^^^^^^^^^^^^^^^^^^^
[WARNING] stderr: AttributeError: 'NoneType' object has no attribute 'text'
